guardian/cdk-cli
# Use gu linting rules

## What does this change?

This PR updates the linting and ts configs to use guardian default values. It also fixes all the errors that the updated config flags up. This fixes #1.

## How to test

Run `yarn lint` and admire the lack of issues. Run `yarn test` to verify that we haven't broken anything. 

## How can we measure success?

The linter is happy 🙂
